_includes/price.html

```diff
@@ -0,0 +1,36 @@
+<section class="flex flex-row md:gap-5 p-2 md:p-10 w-full bg-gray-300 object-cover text-black">
+  <div class="w-full md:w-3/4 p-10 relative p-2 m-auto text-center">
+    <span class="bg-black text-white text-xl px-[12px] py-[8px] rounded-sm">
+      Precio
+    </span>
+    <div class="content-center text-[32px]/[120%] text-center"></div>
+      <h3 class="mt-12 text-2xl w-1/2 m-auto">Disfruta de esta experiencia formativa, vengas de donde vengas. Disponemos de diferentes tarifas adaptadas para cada perfil.</h3>
+      <div class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-4 w-full mt-12">
+        <div class="bg-[#E46199] p-6 rounded-lg border border-black border-solid">
+          <div class="text-7xl font-[Instrument_Serif] mt-2">96€</div>
+          <div class="text-pretty mt-8">Precio por Módulo</div>
+        </div>
+        <div class="bg-[#DF3229] p-6 rounded-lg border border-black border-solid">
+          <div class="text-7xl font-[Instrument_Serif] mt-2">250€</div>
+          <div class="text-pretty mt-8">Módulo común + 1 Módulo de especialización + Experiencias prácticas</div>
+        </div>
+        <div class="bg-[#EE9F2E] p-6 rounded-lg border border-black border-solid">
+          <div class="text-7xl font-[Instrument_Serif] mt-2">480€</div>
+          <div class="text-pretty mt-8">Módulo común + 4 Módulos de especialización + Experiencias prácticas</div>
+        </div>
+    </div>
+    <div class="bg-[#EB834A] p-6 rounded-lg w-full mt-4 flex flex-row border border-black border-solid">
+      <div class="w-3/8 pl-8">
+        <p class="text-left">Menos de</p>
+        <h2 class="text-left text-7xl font-[Instrument_Serif] mt-2">30 años</h2>
```

Comment:
This h2 tags (that aren't really titles, should be bigger, like `text-8xl`, you can take a look in figma and see that it's around 100px, so we have to try to match that with the ones in tailwind.